### PR TITLE
executor: add some missing test suite back

### PR DESCRIPTION
**Pull request by jackysp**
Signed-off-by: Shuaipeng Yu <jackysp@gmail.com>

### What problem does this PR solve? 
Add testSuite4 and seqTestSuite1 back. Maybe we should check if there is another unregisted suite.

### What is changed and how it works?
Add the suites back and fix the tests.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: executor/window_test.go [hunk 1]
```diff
@@ -85,7 +85,7 @@ func (s *testSuite4) TestWindowFunctions(c *C) {
 	result = tk.MustQuery("select row_number() over(rows between 1 preceding and 1 following) from t")
 	result.Check(testkit.Rows("1", "2", "3", "4"))
 	result = tk.MustQuery("show warnings")
-	result.Check(testkit.Rows("Note 3599 Window function 'row_number' ignores the frame clause of window '<unnamed window>' and aggregates over the whole partition"))
+	result.Check(testkit.Rows("Note 3599 Window function 'row_number' ignores the frame clause of window '' and aggregates over the whole partition"))
```
Comment (jackysp) [inline executor/window_test.go]: I'm not sure that if this change is expected or not.
PTAL @winoros

Comment (winoros) [reply executor/window_test.go]: It should be unexpected. Let me take a look.

File: executor/seqtest/seq_executor_test.go [hunk 2]
```diff
@@ -802,10 +925,16 @@ func (c *checkPrioClient) getCheckPriority() pb.CommandPri {
...
 	resp, err := c.Client.SendRequest(ctx, addr, req, timeout)
-	switch req.Type {
-	case tikvrpc.CmdCop:
-		if c.getCheckPriority() != req.Priority {
-			return nil, errors.New("fail to set priority")
+	c.mu.RLock()
+	defer func() {
+		c.mu.RUnlock()
+	}()
+	if c.mu.checkPrio {
```
Comment (eurekaka) [inline executor/seqtest/seq_executor_test.go]: Why do we need this flag?

Comment (jackysp) [reply executor/seqtest/seq_executor_test.go]: Some SQLs in bootstrap and teardown will be set to high priority. But the `getCheckPriority` will always return a normal priority.

**Pull request merged**